src/Tools/CompilerBenchmarks/run-perf.ps1

```diff
@@ -12,18 +12,19 @@ $ErrorActionPreference = "Stop"
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
 try {
-    . (Join-Path $roslynDir "build/scripts/build-utils.ps1")
+    . (Join-Path $roslynDir "eng/common/tools.ps1")
+    . (Join-Path $roslynDir "eng/build-utils.ps1")
 
     # Download dotnet if it isn't already available
     Ensure-DotnetSdk
 
-    $reproPath = Join-Path $binariesDir "CodeAnalysisRepro"
+    $reproPath = Join-Path $ArtifactsDir "CodeAnalysisRepro"
 
     if (-not (Test-Path $reproPath)) {
         $tmpFile = [System.IO.Path]::GetTempFileName()
         Invoke-WebRequest -Uri "https://roslyninfra.blob.core.windows.net/perf-artifacts/CodeAnalysisRepro.zip" -UseBasicParsing -OutFile $tmpFile
         [Reflection.Assembly]::LoadWithPartialName('System.IO.Compression.FileSystem') | Out-Null
```

Comment:
You can use `Unzip` function here defined in tools.ps1: https://github.com/dotnet/roslyn/blob/master/eng/common/tools.ps1#L52